.github/workflows/tiledb-cloud-py.yaml

```diff
@@ -55,6 +63,15 @@ jobs:
             path: ~/.cache/pip
           # - os: macos-12
           # path: ~/Library/Caches/pip
+        exclude:
+          - tiledb-version-spec: "tiledb==0.30"
+            dependencies: fresh
+          - tiledb-version-spec: "tiledb==0.30"
+            is-pr: true
```

Comment:
Builds using the oldest supported tiledb are excluded for unscheduled PR builds.